e2e/images/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM ubuntu:16.04
+ENV container docker
+ENV LC_ALL C
+ENV DEBIAN_FRONTEND noninteractive
+
+#RUN sed -i "s/^exit 101$/exit 0/" /usr/sbin/policy-rc.d
+
+# Install common dependencies
+RUN apt-get -y clean && apt-get -y update && apt-get install -qqyf \
+    ca-certificates \
+    lxc \
+    iptables \
+    jq \
+    software-properties-common \
+    apt-transport-https \
+    curl \
+    git
+
+
+#Delete systemd services we do not need
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done); \
+    rm -f /lib/systemd/system/multi-user.target.wants/*;\
+    rm -f /etc/systemd/system/*.wants/*;\
+    rm -f /lib/systemd/system/local-fs.target.wants/*; \
+    rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
+    rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+    rm -f /lib/systemd/system/basic.target.wants/*;\
+    rm -f /lib/systemd/system/anaconda.target.wants/*;
+
+
+RUN apt-get update &&\
```

Comment:
This line is doing `apt-get update` twice. Perhaps you could setup the apt-sources and then do `apt-get update` once?